pkg/cmd/cli/describe/printer.go

```diff
@@ -957,8 +957,13 @@ func printClusterNetworkList(list *sdnapi.ClusterNetworkList, w io.Writer, opts
 }
 
 func printEgressNetworkPolicy(n *sdnapi.EgressNetworkPolicy, w io.Writer, opts kctl.PrintOptions) error {
-	_, err := fmt.Fprintf(w, "%s\n", n.Name)
-	return err
+	if _, err := fmt.Fprintf(w, "%s\t", n.Namespace); err != nil {
+		return err
+	}
+	if _, err := fmt.Fprintf(w, "%s\n", n.Name); err != nil {
```

Comment:
Simplify...if _, err := fmt.Fprintf(w, "%s\t%s\n", n.Namespace, n.Name); err != nil
egressNetworkPolicyColumns contains only one column 'NAME', do we want to add 'NAMESPACE' column?